packages/protobuf-conformance/src/conformance.ts

```diff
@@ -35,6 +35,9 @@ import {
 } from "./gen/conformance/conformance_pb.js";
 import { fileDesc_google_protobuf_test_messages_proto3 } from "./gen/google/protobuf/test_messages_proto3_pb.js";
 import { fileDesc_google_protobuf_test_messages_proto2 } from "./gen/google/protobuf/test_messages_proto2_pb.js";
+import { fileDesc_google_protobuf_test_messages_edition2023 } from "./gen/google/protobuf/test_messages_edition2023_pb.js";
+import { fileDesc_google_protobuf_test_messages_proto2_editions } from "./gen/google/protobuf/test_messages_proto2_editions_pb.js";
+import { fileDesc_google_protobuf_test_messages_proto3_editions } from "./gen/google/protobuf/test_messages_proto3_editions_pb.js";
```

Comment:
Conformance tests cover editions with two files that are configured to behave the same as the proto2 and proto3 test files, and a separate file specific to editions.

We're passing all tests for JSON and binary serialization, and continue to skip all tests for the text-format, which we do not implement.